src/bundle/Resources/public/scss/_icons.scss

```diff
@@ -93,15 +93,22 @@
             fill: $ez-alloyeditor-color-hover;
         }
     }
+
     .ez-icon {
-        width: 1.5rem;
-        height: 1.5rem;
-        fill: $ez-white;
+        width: 1.25rem;
+        height: 1.25rem;
+        fill: $ez-color-base-medium;
```

Comment:
After testing with your contribution and see there's not enough contrast color we would need to change it to `$ez-black`

Reply:
Icons look like they have a bit of extra space below them.
- Adding to `.ez-btn-ae` flex properties to center them would make them look totally balanced within the toolbar (this would include also _link_ and _unlink_ buttons in _alloyeditor-link-edit_);
- RTEditor buttons (`.ez-btn-ae`) would need to have specific width and height of 34px, instead of the current size they have;